app/models/listing.rb

```diff
@@ -26,6 +23,11 @@ class Listing < ApplicationRecord
   validate :restrict_markdown_input
   validate :validate_tags
 
+  # This is still here because removing it is part of a SEPARATE PR
+  pg_search_scope :search_listings,
+                  against: %i[body_markdown cached_tag_list location slug title],
+                  using: { tsearch: { prefix: true } }
+
```

Comment:
```suggestion
  # This is still here because removing it is part of a SEPARATE PR
  pg_search_scope :search_listings,
                  against: %i[body_markdown cached_tag_list location slug title],
                  using: { tsearch: { prefix: true } }

```